janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -252,6 +252,11 @@
                 "this parameter to true but we do recommend to reindex your indexes and don't use this parameter.",
             ConfigOption.Type.MASKABLE, false);
 
+    public static final ConfigOption<Long> CLIENT_KEEP_ALIVE =
+        new ConfigOption<>(ELASTICSEARCH_NS, "client-keep-alive",
+            "Set a keep-alive timeout (in milliseconds)",
+            ConfigOption.Type.MASKABLE, 900000L);
```

Comment:
I would suggest to not use a default value here. Just remove last parameter (`900000L`).

Reply:
ok, fine for me.